skydb/skydb.go

```diff
@@ -35,14 +35,14 @@ type SkyDB struct {
 // to use "Sia-Agent" as user agent and to get the password for skyd from the
 // environment.
 func New() (*SkyDB, error) {
-	sk, err1 := base64.StdEncoding.DecodeString(os.Getenv("SKYDB_SEC_KEY"))
-	pk, err2 := base64.StdEncoding.DecodeString(os.Getenv("SKYDB_PUB_KEY"))
-	if err1 != nil || err2 != nil {
-		return nil, errors.AddContext(errors.Compose(err1, err2), "failed to decode SKYDB_SEC_KEY and/or SKYDB_PUB_KEY")
-	}
-	if len(sk) == 0 || len(pk) == 0 {
-		return nil, errors.New("missing SKYDB_SEC_KEY or SKYDB_PUB_KEY environment variable")
-	}
+	var entropy [32]byte
+	ent, err := base64.StdEncoding.DecodeString(os.Getenv("SKYDB_ENTROPY"))
```

Comment:
I think this should error if the entropy isn't set. It doesn't seem secure to be using `""`.

Reply:
It does that in the error check below. Although I would also rewrite it to be more clear.

Reply:
It feels redundant to do the two checks separately but if it helps with readability I'm fine with it.